prow/config.yaml

```diff
@@ -13058,6 +13058,21 @@ periodics:
       image: gcr.io/k8s-testimages/kubekins-e2e:latest-master
       imagePullPolicy: Always
 
+- interval: 4h
+  agent: kubernetes
+  name: ci-kubernetes-e2e-kops-aws-encryption-provider
+  labels:
+    preset-service-account: true
```

Comment:
need to change true to "true" as https://github.com/kubernetes/test-infra/commit/1dd7bf44e4d275f9c4d070257d18d8fca7550e10